CONTRIBUTING.md

```diff
@@ -17,23 +17,24 @@ In that case, simply mark the PR with the tag [WIP] (standing for work in progre
 
 # Signing the CLA
 Quickwit is an opensource project licensed a AGPLv3.
-It is also distributed under a commercial license by Quickwit Inc.
+It is also distributed under a commercial license by Quickwit, Inc.
 
 Contributors are required to sign a Contributor License Agreement.
 The process is simple and fast. Upon your first pull request, you will be prompted to
 [sign our CLA by visiting this link](https://cla-assistant.io/quickwit-inc/quickwit).
 
 # Development
 ## Setup & run tests
-1. Install docker https://docs.docker.com/engine/install/
-2. Install localstack https://github.com/localstack/localstack#installing
-3. Install awslocal https://github.com/localstack/awscli-local
-4. Prepare s3 bucket used in tests, execute `./quickwit-cli/tests/prepare_tests.sh`
```

Comment:
`prepare_tests.sh` seems to not be used anymore, you can remove it.

Reply:
It is still used in CI. I will modify the CLI test suite in a follow-up PR, and then I'll get rid of it.